graffiti/graph/traversal/traversal.go

```diff
@@ -122,18 +122,26 @@ func KeyValueToFilter(k string, v interface{}) (*filters.Filter, error) {
 
 		return &filters.Filter{RegexFilter: rf}, nil
 	case *NEElementMatcher:
+		var neFilter *filters.Filter
+		var err error
+
 		switch t := v.value.(type) {
 		case string:
-			return filters.NewNotFilter(filters.NewTermStringFilter(k, t)), nil
+			neFilter = filters.NewNotFilter(filters.NewTermStringFilter(k, t))
 		case bool:
-			return filters.NewTermBoolFilter(k, !t), nil
+			neFilter = filters.NewTermBoolFilter(k, !t)
 		default:
 			i, err := common.ToInt64(t)
 			if err != nil {
 				return nil, err
 			}
-			return filters.NewNotFilter(filters.NewTermInt64Filter(k, i)), nil
+			neFilter = filters.NewNotFilter(filters.NewTermInt64Filter(k, i))
 		}
+		if err != nil {
```

Comment:
I think this is not required since we are setting ```err``` only in default case and have a check there itself.